playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -24,8 +24,6 @@
       openshift_node_image_prep_packages:
       - strace
       - tcpdump
-      - skopeo
```

Comment:
Would like to keep skopeo.

Reply:
it will be there as part of crio build later on. lets not change docker ones for now

Reply:
Skopeo is required to check that images are accessible, so it would installed later on anyway AFAIK